src/features/rewards/walletPanel/index.tsx

```diff
@@ -0,0 +1,229 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+// Helpers
+import { getLocale } from '../../../helpers'
+
+// Styled Components
+import {
+  StyledWrapper,
+  StyledAttentionScore,
+  StyledAttentionScoreTitle,
+  StyledVerified,
+  StyledVerifiedIcon,
+  StyledContainer,
+  StyledScoreWrapper,
+  StyledControlsWrapper,
+  StyledDonateText,
+  StyledVerifiedText,
+  StyledIcon,
+  StyledDonateWrapper,
+  StyledToggleWrapper
+} from './style'
+
+// Components
+import {
+  Grid,
+  Column,
+  Select,
+  Toggle,
+  ButtonSecondary
+} from '../../../components'
+import { Tokens } from '../'
+import Profile, { Provider } from '../profile/index'
+import ToggleTips from '../toggleTips/index'
+
+// Assets
+const alertIcon = require('./assets/alert')
+const monthlyIcon = require('./assets/monthly')
+const verifiedIcon = require('./assets/verified')
+
+export interface Props {
+  id?: string
+  platform?: Provider
+  publisherImg?: string
+  publisherName?: string
+  isVerified?: boolean
+  attentionScore?: string | null
+  donationAction: () => void
+  onToggleTips: () => void
+}
+
+interface State {
+  tipsEnabled: boolean
+  includeInAuto: boolean
+  monthlyAmount: number | null
+}
+
+export default class WalletPanel extends React.PureComponent<Props, State> {
+  constructor (props: Props) {
+    super(props)
+    this.state = {
+      tipsEnabled: true,
+      includeInAuto: true,
+      monthlyAmount: null
+    }
+  }
+
+  onIncludeInAuto = () => {
+    this.setState({
+      includeInAuto: !this.state.includeInAuto
+    })
+  }
+
+  verifiedSubTitle () {
+    if (!this.props.isVerified) {
+      return null
+    }
+
+    return (
+      <StyledVerified>
+        <StyledVerifiedIcon>
+          {verifiedIcon}
+        </StyledVerifiedIcon>
+        <StyledVerifiedText>
+          {getLocale('verifiedPublisher')}
+        </StyledVerifiedText>
+      </StyledVerified>
+    )
+  }
+
+  publisherInfo () {
+    return (
+      <StyledWrapper>
+        <Profile
+          type={'big'}
+          title={'Bart Baker'}
+          provider={this.props.platform}
+          src={this.props.publisherImg}
+          subTitle={this.verifiedSubTitle()}
+        />
+      </StyledWrapper>
+    )
+  }
+
+  getSelectTheme () {
+    return {
+      border: 'none',
+      padding: '0px',
+      margin: '3px 0px 0px 0px',
+      background: 'inherit'
+    }
+  }
+
+  donationDropDown () {
+    return (
+      <div>
```

Comment:
Let's fix margin for select with this div, so that we don't need to add new theme props to select